src/arithmetic/path_funcs/path_funcs.c

```diff
@@ -192,9 +189,11 @@ SIValue AR_SHORTEST_PATH(SIValue *argv, int argc) {
 	}
 
 	// Invoke the BFS algorithm
-	res = LAGraph_bfs_pushpull(&V, &PI, ctx->R, ctx->TR, src_id,
-							   &dest_id, max_level, true);
+	res = LG_BreadthFirstSearch_SSGrB(&V, &PI, ctx->R, ctx->TR, src_id,
```

Comment:
Do we need both `PI` and `V` to compute `AR_SHORTEST_PATH` ?
Seems like `V` is used to determine the path length `src -> dest` but there are other ways to figure out the length,
if skipping on computing `V` is possible in this context I think we should, see implications in `LG_BreadthFirstSearch_SSGrB`